### PR TITLE
CLD-263-update-page-designer-user-agent

### PR DESCRIPTION
**Pull request by asad-rafter**

### CODE REVIEW

File: cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+var System = require('dw/system/System');
+
+module.exports.cloudinaryConstants = {
+    CLD_TRACKING_PARAM: '?_i=AH',
```
Comment (yuval-cloudinary) [inline cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js:4]: This seems to have leaked from cld-264, please remove

File: cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    API_TRACKING_PARAM: 'CloudinarySalesForcePageDesigner/20.0.4 (CommerceCloud ' + System.compatibilityMode + ') Architecture SFCC'
```
Comment (yuval-cloudinary) [inline cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js:5]: We requested the following user-agent:
CloudinarySalesForcePageDesigner/<cartridge ver> (<platform> <ver>) <SFCC arch>

Please have the cartridge version and the architecture taken from the environment (not hard-coded)

**Review by yuval-cloudinary — CHANGES_REQUESTED** [review]

File: cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js (hunk 2 above)
Comment (asad-rafter) [reply cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js:5]: do you want me to create custom preference for this?

Comment (yuval-cloudinary) [reply cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js:5]: This is not a user-preference, so please keep it in the code.
However, I expect the cartridge version to reflect the cartridge version, not a hard-coded 20.0.4.
Similarly, the system architecture is SFRA/SG/PWA/etc... not a hard-coded "Architecture SFCC"

Comment (asad-rafter) [reply cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js:5]: From where do you want me to get the cartridge version value and system architecture?

**Comment by yuval-cloudinary** [comment]
The caryridge has the version in multiple places, please use the most
appropiate. For the architecture, maybe SFCC docs?

File: cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js (hunk 2 above)
Comment (asad-rafter) [reply cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js:5]: For the cartridge version we have fixed that, but for SFCC architecture there is no way in SF to get the site architecture if its SFRA/SG/PWA, the only thing we can do is to make a site preference and set and read value from there.

Comment (yuval-cloudinary) [reply cartridges/int_cloudinary_pd/cartridge/experience/utils/constants.js:5]: @asad-rafter I see. Please put there a hard-coded "SFPD", e.g.:
CloudinarySalesForcePageDesigner/\<cartridge ver> (\<platform> \<ver>) SFPD

**Review by yuval-cloudinary — APPROVED** [review]

**Review by yuval-cloudinary — APPROVED** [review]

**Pull request merged**